tools/edl

```diff
@@ -30,15 +30,41 @@ if [ "$(< .git/HEAD)" != "$VERSION" ]; then
 fi
 popd > /dev/null
 
-$EDL_PATH/edl "$@"
+if [[ "$(uname)" == 'Darwin' ]]; then
+  ARCH=$(uname -m)
+
+  # install brew if required
```

Comment:
I know openpilot does it, but installing brew is pretty invasive (plus we don't want to have maintain this). Let's just put "fresh" setup instructions in the README